[Marketplace Contribution] PAN-OS by Palo Alto Networks - Content Pack Update (#28156)
demisto/content: Packs/PAN-OS/ReleaseNotes/1_17_20.md

#### Integrations

##### Palo Alto Networks PAN-OS

> - Added the command ***pan-os-list-device-groups*** to get all of the device groups names.

Reviewer: ```suggestion
- Added the ***pan-os-list-device-groups*** command which retrieves all the device groups in Panorama.
```
